manipulate-strings/app.js

```diff
@@ -0,0 +1,4 @@
+let firstName = 'Paul';
+let lastName = 'Banjawan';
+let role = 'developer';
```

Comment:
`const` instead of `let`
```suggestion
const firstName = 'Paul';
const lastName = 'Banjawan';
const role = 'developer';
```

Reply:
Hi @xianvalencia , this has already been resolved